python/example.py

```diff
@@ -47,18 +47,27 @@
 viewer.add(b, name='b')
 
 #Third layer (points)
-viewer.add(data=[], name='layer_point', layer_type='point')
+viewer.add(data=[[5,8,10],[30,23,21]], name='layer_point', layer_type='point')
 
 #Fourth layer (synapses)
-viewer.add(data=[], name='layer_synapse', layer_type='synapse')
+# the 1D vector of coordinates will be decomposed to coordinate pairs, so you
+# need some duplicated nodes to define a tree
+viewer.add(data=[[0,0,0],[1,3,5], [3,6,9],[5,34,24]],
+                               name='layer_synapse', layer_type='synapse')
 
 #Fith layer (vector graphics)
 viewer.add(data=[[10, 20 ,30],
                  [15, 20 ,30],
                  [15, 25 ,30]], name='layer_vector_graphics', layer_type='line')
 
+# Sixth layer (skeleton)
+skeleton = Skeleton([[10, 20, 30],
+                     [15, 20, 30],
+                     [15, 25, 30]], [[1,2],[2,3]])
+#viewer.add(name='skeleton', layer_type='single_mesh', data = skeleton)
```

Comment:
Why comment it out?

Reply:
this test do not work, I don't know the correct format.

Reply:
just removed the skeleton layer since the example do not work.